fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/MicroProfileJWTAuthFraction.java

```diff
@@ -68,7 +68,7 @@
     @Configurable("thorntail.microprofile.jwt.realm")
     @Configurable("thorntail.microprofile.jwtauth.realm")
     @AttributeDocumentation("If set, a security domain with this name that supports MicroProfile JWT is automatically created in the security subsystem."
-                            + " The realmName parameter of the @LoginConfig annotation must be set to the same value.")
+                            + " If the @LoginConfig realName property is configured then it must have the same value.")
```

Comment:
Typo, `realName` -> `realmName`.

Reply:
Oh and the text should probably also mention that this realm will automatically be used for JWT auth.

Reply:
@Ladicek the 1st part of the doc already says so or would you like to expand/change it ? Something like `MicroProfile JWT realm name. If set, a security domain with this name is automatically created in the security subsystem.` ?

Reply:
The text says that a security domain is automatically created. But with this PR, not only is it automatically created, but is also automatically used (and in fact must be used). Do I make sense? :-)

Reply:
But this is the case even now on the master ? This PR only allows to have LoginConfg realmName be empty

Reply:
On current master, if you set this option, a security domain will be automatically created, but you still need to configure it in `@LoginConfig(realmName)`. With this PR, you no longer need to set `@LoginConfig(realmName)`.